README.md

```diff
@@ -1 +1,8 @@
-# create202404
\ No newline at end of file
+# create202404
+Some description that doesnt mean anything.
+
+## Section 2
+How to enable something that need to be enabled without the need to enable it.
+
```

Comment:
The logical order of sections in the document is incorrect. Please reorder them to maintain a coherent flow of content.